libs/openFrameworks/video/ofVideoPlayer.cpp

```diff
@@ -77,7 +77,7 @@ bool ofVideoPlayer::load(string name){
         if(bUseTexture){
         	if(player->getTexturePtr()==nullptr){
 				if(tex.empty()) {
-					tex.resize(max(player->getPixels().getNumPlanes(),1));
```

Comment:
Oh, whoops, I noticed you changed it back to the define, rather than the std::max.  I'd try to use the standard library std::min/max as I believe we're trying to cut down on our custom defines ...